views/insight_fallback.tmpl

```diff
@@ -0,0 +1,23 @@
+{{define "insight_fallback"}}
+<!DOCTYPE html>
+<html lang="en">
+
+{{template "html-head" "Insight Fallback Page"}}
+    {{template "navbar" . }}
+    <div class="container main">
+        <div>
+            <h2 style="text-align: center; margin-top: 0px">Looking for the Insight API?</h2>
+            <p style="text-align: center; margin-bottom: 5px">
+                The Insight API is available under the API sub-path. (e.g. <a href="{{.RequestURI}}/api/status">{{.RequestURI}}/api/status</a>).
```

Comment:
Using RequestURI for the same reason.  TODO: see how this interacts with a reverse proxy, which makes it's own requests, and attempt to get the request path as it hits the proxy from the client if necessar.